### DISCUSSION TITLE
Can you explain what all things makes this "perfect programming language" :-))

### DISCUSSION THREAD

**Discussion by Greatz08** [TodePond/GulfOfMexico#700] [General]
I have seen syntax changes and some simpler methods used for some operations but still i cant find too many differences because i am not that much experienced and many are like me so please not just for me but for everyone  explain all things which make this "perfect programming language" as been mentioned so that it becomes easier to understand this point  :-))

**Comment by TodePond** [comment]
it's because of ai

**Comment by eru123** [comment]
Everyone claims to be, so do they.

**Comment by loglot** [comment]
because it is just true, it's the best!, the title "perfect programming language" is absolutely not satire
